plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineStandardWtpProject.java

```diff
@@ -111,10 +110,14 @@ static void installAppEngineRuntime(IFacetedProject project, IProgressMonitor mo
     Set<IProjectFacetVersion> facets = new HashSet<>();
     facets.add(WebFacetUtils.WEB_25);
     Set<IRuntime> runtimes = RuntimeManager.getRuntimes(facets);
-    IRuntime appEngineRuntime = RuntimeManager.getRuntime("App Engine");
-    
     project.setTargetedRuntimes(runtimes, monitor);
-    project.setPrimaryRuntime(appEngineRuntime, monitor);
+    
+    try {
+      IRuntime appEngineRuntime = RuntimeManager.getRuntime("App Engine");
```

Comment:
`RuntimeManager.isRuntimeDefined("App Engine")`?

Does anything bad happen if we don't have a primary runtime?  I'd guess not, since that's no different from the situation where we didn't set a runtime at all.